ink/goddess/reward_joi

```diff
@@ -345,7 +345,7 @@ Can you keep continue without orgasming?
 {shuffle:
     - -> slow ->
     - -> fast ->
-    - -> squeeze ->
+    - {hasPenis: ->squeeze->|->->}
```

Comment:
Will you write what happens when a person is told to squeeze their lips? Or would you prefer I add that? :)